src/scala/com/github/johnynek/bazel_deps/DepsModel.scala

```diff
@@ -484,16 +486,18 @@ object Language {
 
   case object Java extends JavaLike {
     def asString = "java"
+    def asReversableString = asString
   }
 
   case object Kotlin extends JavaLike {
     def asString = "kotlin"
-
+    def asReversableString = asString
   }
 
   case class Scala(v: Version, mangle: Boolean) extends Language {
     def asString = if (mangle) "scala" else "scala/unmangled"
     def asOptionsString: String = s"scala:${v.asString}"
+    def asReversableString = s"scala:$mangle:${v.asString}"
```

Comment:
I think using `s"$asString:${v.asString}"` would be nicer to read. `$mangle` will produce a boolean, which will be confusing to read.

Reply:
👍